TotalCrossVM/src/nm/ui/android/skia.cpp

```diff
@@ -222,7 +222,11 @@ int skia_makeBitmap(int32 id, void *data, int32 w, int32 h) {
 
     if (id < 0) { // must create a new bitmap
         SkBitmap bitmap;
-        bitmap.installPixels(SkImageInfo::Make(w, h, kN32_SkColorType, kUnpremul_SkAlphaType), (void*)converted, sizeof(Pixel) * w, releaseProc, nullptr);
+		bitmap.installPixels(SkImageInfo::Make(w, h, kN32_SkColorType, kUnpremul_SkAlphaType),
```

Comment:
Unrelated changes.

Reply:
Those are actually related! The releaseProc was changed to be able to also handle freeing dstPixels, which is allocated as a void* instead of an integer type.